networking/podenv.go

```diff
@@ -218,6 +222,11 @@ func copyFileToDir(src, dstdir string) (string, error) {
 }
 
 func loadUserNets(localConfig string, netsLoadList common.NetList) ([]activeNet, error) {
+	if netsLoadList.None() {
+		log.Printf("Networking namespace with loopback only")
```

Comment:
Digging a little deeper, "\n" is definitely not needed: https://github.com/golang/go/blob/master/src/log/log.go#L163